foreman/data_refinery_foreman/surveyor/transcriptome_index.py

```diff
@@ -65,7 +60,7 @@ def __init__(self, species: Dict):
         self.url_root = "ensemblgenomes.org/pub/release-{assembly_version}/{short_division}"
         self.short_division = DIVISION_LOOKUP[species["division"]]
         self.assembly = species["assembly_name"].replace(" ", "_")
-        self.assembly_version = OTHER_DIVISIONS_ASSEMBLY_VERSION
+        self.assembly_version = requests.get(DIVISION_RELEASE_URL).json()["release"]
```

Comment:
Use the retrier in util?

I wish `requests` came with a warning (it actually does in the upcoming v3): NEVER USE .GET IN PRODUCTION WITHOUT A `,timeout=`! The retrier we already have will take care of this.